.pipelines/cni/singletenancy/cniv2-template.yaml

```diff
@@ -104,7 +114,11 @@ stages:
                   make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(commitID)
                   kubectl cluster-info
                   kubectl get po -owide -A
-                  sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(make cns-version) CNI_VERSION=$(make cni-version) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO)
+                  if [ "${{parameters.os}}" == "windows" ]; then
+                    sudo -E env "PATH=$PATH" make test-load CNS_ONLY=true CNS_VERSION=$(make cns-version) CNI_VERSION=$(make cni-version) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO)
+                  else
+                    sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(make cns-version) CNI_VERSION=$(make cni-version) INSTALL_CNS=true INSTALL_AZURE_CNI_OVERLAY=true CNS_IMAGE_REPO=$(CNS_IMAGE_REPO)
```

Comment:
What's the difference between `test-integration` and `test-load` ? I thought we replaced one with another

Reply:
Within the PR pipeline we are going to be utilizing test load to run core scenarios ( CNS Restart, scaling, validation) and integration will be used for the more through functional test. Due to the requirements of goldpinger, we need bigger VM's and as such it should be used in the CNI Release Test Pipeline.

For the block in question. We have not done the work yet to bring over windows goldpinger, so we are currently just using test load for CNS installation and test integration as intended.